examples/circom.rs

```diff
@@ -89,6 +90,16 @@ impl<F: LurkField> CircomGadget<F> for CircomSha256<F> {
         );
         s.intern_num(expected)
     }
+
+    fn simple_evaluate_lem(&self, _s: &LEMStore<F>, _args: &[LEMPtr<F>]) -> LEMPtr<F> {
+        // TODO: actually use the lurk inputs
```

Comment:
I believe this is the third variant of this comment in the file. Please open an issue explaining what should happen and reference it here.